src/assets/data/edition/series/2/section/2a/m46/svg-sheets.json

```diff
@@ -0,0 +1,225 @@
+                        "image": "",
+                        "partial": "b",
+                        "convolute": ""
+                    }
+                ]
+            }
+        ],
+        "sketchEditions": [
+            {
+                "id": "M_46_Sk1",
+                "label": "M 46 Sk1",
+                "content": [
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m46/M46_Sk1-1von2-final.svg",
+                        "image": "",
+                        "partial": "a",
+                        "convolute": "A"
+                    },
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m46/M46_Sk1-2von2-final.svg",
+                        "image": "",
+                        "partial": "b",
+                        "convolute": "A"
+                    }
+                ]
+            },
+            {
+                "id": "M_46_Sk2",
+                "label": "M 46 Sk2",
+                "content": [
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m46/M46_Sk2-1von2-final.svg",
+                        "image": "",
+                        "partial": "a",
+                        "convolute": "A"
+                    },
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m46/M46_Sk2-2von2-final.svg",
+                        "image": "",
+                        "partial": "b",
+                        "convolute": "A"
+                    }
+                ]
+            },
+            {
+                "id": "M_46_Sk3",
+                "label": "M 46 Sk3",
+                "content": [
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m46/M46_Sk3-1von5-final.svg",
+                        "image": "",
+                        "partial": "a",
+                        "convolute": "A"
+                    },
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m46/M46_Sk3-2von5-final.svg",
+                        "image": "",
+                        "partial": "b",
+                        "convolute": "A"
+                    },
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m46/M46_Sk3-3von5-final.svg",
+                        "image": "",
+                        "partial": "c",
+                        "convolute": "A"
+                    },
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m46/M46_Sk3-4von5-final.svg",
+                        "image": "",
+                        "partial": "d",
+                        "convolute": "A"
+                    },
+                    {
+                        "svg": "assets/img/edition/series/2/section/2a/m46/M46_Sk3-5von5-final.svg",
```

Comment:
Diese Datei (5von5) fehlt, allerdings auch kein 5. Eintrag in Inhaltsangabe. Sind es nur 4 Skizzeneinheiten in Sk3?